crates/sui-json-rpc-types/src/sui_extended.rs

```diff
@@ -42,6 +47,17 @@ pub struct EpochInfo {
     pub reference_gas_price: Option<u64>,
 }
 
+impl EpochInfo {
+    pub fn committee(&self) -> Committee {
```

Comment:
Can we return Result<Committee> instead of unwrapping AuthorityName::from_bytes?